content/get-started/sidebar.ts

```diff
@@ -73,6 +73,7 @@ export const sidebar: SidebarDefinition = [
               "cli-cmd-wallet-multisig-commitment-aggregate",
               "cli-cmd-wallet-multisig-signature-create",
               "cli-cmd-wallet-multisig-signature-aggregate",
+              "cli-cmd-wallet-multisig-dkg-round1",
```

Comment:
should we have the dkg commands under an additional `dkg` dropdown?

Reply:
Yeah good call